New section and small edits (#51)
jessegmeyerlab/proteomics-tutorial: content/03.biochemistry-basics.md

## Biochemistry Basics {.page_break_before}

### Proteins

Proteins are large biomolecules or biopolymers made up of amino acids which are linked by peptide bonds. 
They perform various functions in living organisms ranging from having structural roles to functional involvement in cellular signaling and the catalysis of chemical reactions (enzymes). 
Proteins are made up of 20 different amino acids (not counting pyrrolysine and selenocysteine, which only occur in specific organisms) and their sequence is encoded in their corresponding genes.

> The human genome encodes more than 20,000 different proteins of varying abundance. 

Reviewer: ```suggestion
The human genome encodes more than 20,000 different proteins.
Each protein is present at a different abundances.
```

Previous studies have shown that the concentration range of proteins can span over a range of at least seven orders of magnitude to up to 20 000 000 copies per cell and that their distribution is tissue-specific[@DOI:10.1038/msb.2011.82][@DOI:10.1016/j.cell.2020.08.036].
Due to genetic variations, as well as alternative splicing and post-translational modifications, multiple different proteoforms can be produced from one single gene[@DOI:10.1038/nmeth.2369].

#### PTMs

> Proteins can be post-translationally modified through enzymatic and non-enzymatic reactions _in vivo_ and _in vitro_. 

Reviewer: ```suggestion
Proteins can be post-translationally modified through enzymatic and non-enzymatic reactions _in vivo_ and _in vitro_ [@DOI:10.1093/database/baab012]. 
```

Post-translational modifications (PTMs) can be reversible or non-reversible covalent modifications of amino acids.
The most commonly studied and biologically relevant post-translational modifications include ubiquitinations (Lys, Cys, Ser, Thr, N-term), succinylations (Lys), methylations (Arg, Lys, His, Glu, Asn, Cys), disulfide bonds (Cys-Cys), oxidations (Met, Trp, His, Cys), phosphorylations (Ser, Thr, Tyr, His), acetylations (Lys, N-term), glycosylations (Arg, Asp, Cys, Ser, Thr, Tyr, Trp) and lipidations.
Post-translational modification of a protein can alter its function, activity, structure, location and interactions. 

> Deregulation of PTMs is linked to cellular stress and diseases.[@DOI:10.1038/s41570-020-00223-8]

Reviewer: ```suggestion
Deregulation of PTMs is linked to cellular stress and diseases[@DOI:10.1038/s41570-020-00223-8].
```

#### Protein Structure

> Almost all proteins (except for intrinsically disordered proteins) fold into 3D structures either by themselves or assisted through chaperones[@DOI:10.1007/s12013-021-00970-5]. 

Reviewer: ```suggestion
Almost all proteins (except for intrinsically disordered proteins[@DOI:10.1146/annurev-biochem-072711-164947]) fold into 3D structures either by themselves or assisted through chaperones[@DOI:10.1007/s12013-021-00970-5]. 
```
